pkgs/applications/audio/uhhyou.lv2/default.nix

```diff
@@ -0,0 +1,76 @@
+{ stdenv
+, fetchgit
+, fftw
+, libGL
+, libX11
+, libjack2
+, liblo
+, lv2
+, pkgconfig
+}:
+
+stdenv.mkDerivation rec {
+  pname = "uhhyou.lv2";
+  version = "unstable-2020-05-27";
+
+  src = fetchgit {
+    url = "https://github.com/ryukau/LV2Plugins";
+    rev = "f20d1c8a624d3a757f820b5c41c70f6f9bb6dc22";
+    deepClone = true;
+    sha256 = "12g9j4rrhkw91sbnqi8xg8wkl5j34m0l7fdwhqa2ny526n52lq7k";
+  };
```

Comment:
Oh, silly me I didn't notice that before - Use `fetchFromGitHub`.

Reply:
That doesn't have deepclone.

Reply:
It has `fetchSubmodules = true;`.

Reply:
Indeed.  :)